tree/treeplayer/src/TTreeProxyGenerator.cxx

```diff
@@ -29,9 +29,9 @@
 
   When using in interpreted mode understand why the reloading reloads the calling script and then crashes :(
 
-  CINT does not properly call the custom operators when doing return fNtrack.
+  Cling does not properly call the custom operators when doing return fNtrack.
 
-  CINT does not handle fMatrix[2][1] well.
+  Cling does not handle fMatrix[2][1] well.
```

Comment:
Also here, I would maybe keep CINT, unless you can confirm Cling also doesn't handle that well.